Inspicio/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -107,9 +107,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Description");
 
-                    b.Property<int>("NoOfDislikes");
+                    b.Property<int>("NoOfApprovals");
```

Comment:
Calculating this `NoOfApprovals` value when needed, rather than storing it in the DB should also make the code a bit simpler as well - but it's the usual trade off of perf (more DB calls) vs maintainability (simpler code).

[[+_____]](https://gist.github.com/WilkaH/29df175966a76687f846)